packages/dialog/stories/dialog-wrapper.stories.ts

```diff
@@ -23,6 +23,7 @@ import '@spectrum-web-components/overlay/overlay-trigger.js';
 import '@spectrum-web-components/dialog/sp-dialog-wrapper.js';
 import { landscape } from './images.js';
 import { overlayTriggerDecorator } from './index.js';
+import { DialogWrapper } from '../src/DialogWrapper.js';
```

Comment:
will this work or do we need to do like this?
`@spectrum-web-components/alert-dialog/src/DialogWrapper.js`

Reply:
Nice catch!!